Shaders/Common/public/BasicStructures.fxh

```diff
@@ -137,12 +137,25 @@ struct CameraAttribs
     matrix mProjInv;
     matrix mViewProjInv;
 #endif
+    uint iFrameIndex;
+    float fDummy0;
+    float fDummy1;
+    float fDummy2;
 
-    float4 f4ExtraData[5]; // Any appliation-specific data
+    float4 f4ExtraData[4]; // Any appliation-specific data
```

Comment:
⚠️ The array size can't be changed to 4, some shaders need all 5 elements

Reply:
I reverted CameraAttibs to its initial state.